myia/abstract/data.py

```diff
@@ -63,15 +63,6 @@ def __init__(self, options):
         # list is different.
         super().__init__(OrderedSet(options))
 
-    def _serialize(self):
```

Comment:
You should remove the `@serializable` tag from the class too.

Reply:
Ah, yes, good catch.

Reply:
Done.